taccsite_cms/contrib/taccsite_data_list/templates/data_list.html

```diff
@@ -0,0 +1,21 @@
+{% load cms_tags %}
+
+{% if instance.type_style == 'dlist' %}
+
+<dl {# class="c-data-list" #}{{ instance.attributes_str }}>
```

Comment:
__Explanation:__ The `class="c-data-list"` is added via `cms_plugins.py` so that only `instance.attributes_str` adds the class (thus only one class is added; if two, then one could be ineffectual).

The comment is here so that template authors know the markup will appear, but without verbose comment, because I don't want to write it for every plugin's template.

__Suggestions welcome.__

Reply:
I think that's ok!

Reply:
Cool, thanks.